java/src/main/java/org/extism/sdk/manifest/MemoryOptions.java

```diff
@@ -0,0 +1,11 @@
+package org.extism.sdk.manifest;
+
+public record MemoryOptions(
+
+        /**
+         * Max number of pages.
+         */
+        Integer max // Integer to allow `null` as max
```

Comment:
It's yet to be release, but we recently renamed this to `max_pages` for better clarity. 

We could do this now, and align the release with an upcoming Extism release (though `max` will still work as an alias), or we could release the Java SDK with this as `max` and follow up with another release shortly thereafter with the new name. 

@bhelx @zshipko, any thoughts?

Reply:
I’m open to either. Just depends on if we want to make a release as soon as possible or wait for the next runtime release.

Reply:
Also open to either, but my opinion is I'd prefer to wait until we have a few breaking changes grouped up. Unless we have a rush to get it out.

Reply:
Ok, I agree -- we can ship it with this as-is and release an update when we are ready. Thanks!